src/Start.js

```diff
@@ -44,5 +45,23 @@ const StyledContainer = styled.div`
 `
 
 const StyledTextContainer = styled.div`
-  flex: 1;
+  padding: 25px 0;
+`
+
+const StyledParagraphHeadline = styled.p`
```

Comment:
If it is a headline, why not use h1, h2, h3 ... or h6 instead of a paragraph element?